README.md

```diff
@@ -60,7 +60,7 @@ Instructions for setting up Serge on Kubernetes can be found in the [wiki](https
 | **Alfred** | 40B |
 | **CodeLLaMA** | 7B, 13B |
 | **Falcon** | 7B, 7B-Instruct, 40B, 40B-Instruct |
-| **LLaMA 2**  | 7B, 13B, 70B, 70B-Chat, 70B-OASST |
```

Comment:
Why? `-chat` has more downloads than `oasst`

Reply:
Sorry for the mistake